x-pack/plugins/secops/public/mock/kibana_config.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { AppFrameworkAdapter } from '../lib/lib';
+
+export const mockFrameworks: { [name: string]: Partial<AppFrameworkAdapter> } = {
+  default_ET: {
+    dateFormat: 'MMM D, YYYY @ HH:mm:ss.SSS',
+    dateFormatTz: 'America/New_York',
+  },
+  default_MT: {
+    dateFormat: 'MMM D, YYYY @ HH:mm:ss.SSS',
+    dateFormatTz: 'America/Denver',
+  },
+  default_UTC: {
+    dateFormat: 'MMM D, YYYY @ HH:mm:ss.SSS',
+    dateFormatTz: 'UTC',
+  },
+};
```

Comment:
Optional. You might be able to use Record + Readonly if you want to. It's more fashionable in the TypeScript world if you choose to.

```sh
export const mockFrameworks: Readonly<Record<string, Partial<AppFrameworkAdapter>> = {
```

But it's not required and I didn't test it out locally. It's just typically more readable than the name string biz.

Reply:
Oh that's great -- will use, thanks! :)